core/src/main/java/org/stellar/anchor/sep24/Sep24Transaction.java

```diff
@@ -236,6 +236,18 @@ public interface Sep24Transaction extends SepTransaction {
 
   String getMessage();
 
+  String getQuoteId();
```

Comment:
We would also need to add this field to the platform transaction. It's being mapped here https://github.com/stellar/java-stellar-anchor-sdk/blob/8e595eab7aa01147d610cb39313afa0c18d75630/core/src/main/java/org/stellar/anchor/util/TransactionHelper.java#L172-L196

Reply:
The docs would need to be updated to include `quote_id` under the SEP-24 transaction as well. https://developers.stellar.org/api/anchor-platform/resources/get-transaction